cmake/scripts/common/AddonHelpers.cmake

```diff
@@ -255,8 +257,7 @@ macro (build_addon target prefix libs)
 
         if(CMAKE_BUILD_TYPE MATCHES Debug)
           # for debug builds also install the PDB file
-          get_filename_component(LIBRARY_DIR ${LIBRARY_LOCATION} DIRECTORY)
-          install(FILES ${LIBRARY_DIR}/${target}.pdb DESTINATION ${target}
+          install(FILES ${PROJECT_BINARY_DIR}/${CMAKE_BUILD_TYPE}/${target}.pdb DESTINATION ${target}
                   COMPONENT ${target}-${${prefix}_VERSION})
         endif()
```

Comment:
@fetzerch, @hudokkow or @paxxi
The zip packaging is broken if it is used directly within Visual Studio under Windows. This patch fixes the issue for me. Does one of you have the time to do further tests?